submit.py

```
@@ -0,0 +1,53 @@
+        "jira",
+        "issue",
+        "create",
+        "--no-input",
+        f"-t{data['type']}",
+        f"-s{data['summary']}",
+        "--template",
+        template,
+    ]
+    print(command)
+
+    result = subprocess.run(command, capture_output=True)
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)